add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
- Observations: The script directly accesses the $_POST superglobal and HTML-form elements are not validated or sanitized, which could lead to SQL injection or XSS attacks.
- Recommendations: Always validate and sanitize all inputs from user forms before using them in your logic.

Example:
```php
$title = filter_var($_POST['title'], FILTER_SANITIZE_STRING);
$author = filter_var($_POST['author'], FILTER_SANITIZE_STRING);
$year = filter_var($_POST['year'], FILTER_VALIDATE_INT);
```